### ISSUE TITLE
Highcharts needs to support 508 compliance

### ISSUE THREAD

**Issue by vardhini** [highcharts/highcharts#2125]
Highcharts is not 508 compliant (https://www.section508.gov)
There are W3C guidelines for accessibility features in SVG (http://www.w3.org/TR/SVG-access/) which could help in solving this. We cannot ship our software with highcharts when compliance is mandatory and we would love to see this address sooner than later.

**Comment by oysteinmoseng** [comment]
Closing, see https://www.highcharts.com/docs/chart-concepts/accessibility
